client/app/scripts/superdesk-archive/module.js

```diff
@@ -217,7 +217,7 @@ define([
          * and Killed.
          *
          * @param {Object} item
-         * @return true if the state of the item is in one of the published states, false otherwise.
+         * @return boolean if the state of the item is in one of the published states, false otherwise.
```

Comment:
The editor `pycharm` I use was complaining about it and used one of the options given by it. Also jshint and eslint didn't complain about it. So left it like that.